firefox-ios/Client/Frontend/Onboarding/Views/OnboardingMultipleChoiceButtonView.swift

```diff
@@ -39,19 +39,19 @@ class OnboardingMultipleChoiceButtonView: UIView, Themeable {
         imageView.image = UIImage(named: self.viewModel.info.imageID)
         imageView.layer.cornerRadius = UX.Measurements.imageCornerRadius
         imageView.layer.borderWidth = UX.Measurements.imageBorderWidth
-        imageView.layer.borderColor = UIColor.clear.cgColor
         imageView.accessibilityIdentifier = "\(self.viewModel.a11yIDRoot)ImageView"
         imageView.isAccessibilityElement = false
     }
 
     private lazy var titleLabel: UILabel = .build { label in
-        label.numberOfLines = 0
         label.textAlignment = .center
         label.adjustsFontForContentSizeCategory = true
-        label.font = DefaultDynamicFontHelper.preferredBoldFont(withTextStyle: .body, size: 13)
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body, size: 13)
```

Comment:
Although `DefaultDynamicFontHelper` was used prior, while we are here, what do you think about replacing it with `FXFontStyles` instead?

Reply:
Oh yes, totally forgot.